spec/models/application_choice_spec.rb

```diff
@@ -340,18 +351,18 @@
           .to change(application_choice, :provider_ids).to(expected_ids)
       end
 
-      it 'updates provider_ids if the original course is updated' do
-        original_course = create(:course, :with_accredited_provider)
-        original_course_option = create(:course_option, course: original_course)
+      it 'updates provider_ids if the course is updated' do
```

Comment:
could we also add a new test to check that we set original course ids if all three courses are different?

Reply:
do we want to start reading `original_course_option` in `provider_ids_for_access` then?